script.js

```diff
@@ -1,4 +1,87 @@
+var gameMode = 'player1rolls';
+var player1Dice1;
+var player1Dice2;
+var player1Pick;
+var player2Dice1;
+var player2Dice2;
+var player2Pick;
+
+var diceRoll = function () {
+  // number from 0 to 1
+  var digit = Math.random();
+  // number from 0 to 6
+  var number = digit * 6;
+  // integer to float from 0 to 5
+  var integer = Math.floor(number) + 1;
+  return integer;
+};
+
+var player1turn = function () {
+  player1Dice1 = diceRoll();
+  player1Dice2 = diceRoll();
+  gameMode = 'player1picks';
+};
+
+var player2turn = function () {
+  player2Dice1 = diceRoll();
+  player2Dice2 = diceRoll();
+  gameMode = 'player2picks';
+};
+
+var player1P = function (input) {
```

Comment:
Consider using more descriptive function names. Fn names should start with a verb for clarity